### PR TITLE
Add note about log length in the web console

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1723895

Issue started by customer via
https://access.redhat.com/support/cases/#/case/02400963

### CODE REVIEW

**Comment by mburke5678** [comment]
@xltian Please take a look

**Comment by xltian** [comment]
@yapei please review from QE, thanks

File: install_config/master_node_configuration.adoc [hunk 1]
```diff
@@ -1763,6 +1763,12 @@ $ openshift start node --config=/openshift.local.config/node-<node_hostname>/nod
 
 {product-title} collects log messages for debugging, using the `systemd-journald.service` for nodes and a script, called `master-logs`, for masters.
 
+[NOTE]
+====
+The number of lines displayed in the web console is hard-coded at 5000 and cannot be changed.
```
Comment (yapei) [inline install_config/master_node_configuration.adoc]: Yes, that's the truth. The content looks good to me

**Comment by yapei** [comment]
LGTM from QE's point of view

**Review by spadgett — APPROVED** [review]
lgtm

**Comment by spadgett** [comment]
Just confirming this is 3.x doc only? IIRC, the 4.x console has a 1000 line limit.

**Comment by mburke5678** [comment]
@yapei @spadgett Thank you for the quick review!

**Comment by mburke5678** [comment]
@spadgett This PR is for 3.x only. I can add a similar note with 1000 lines to 4.x  using a separate PR.

**Pull request merged**